README.md

```diff
@@ -4,54 +4,105 @@
+## Overview
+
+Bulldog is an opinionated base library for building .NET command line tools. It provides a robust foundation that handles common concerns in CLI tool development, allowing developers to focus on implementing their core business logic rather than boilerplate setup.
+
+## Features
+
+- **Logging Integration**: Built-in Serilog configuration for structured logging
+- **Dependency Injection**: Automatic DI container setup and configuration
+- **Cancellation Handling**: Graceful handling of cancellation tokens and signals
+- **Command Line Parsing**: Integration with CommandLineParser library for robust argument handling
+- **Standardized Structure**: Enforces best practices while maintaining flexibility
+- **Extensible Design**: Provides extension points for customization
 
 ## Prerequisites
 
 * .NET 7.0 SDK
 * nuke global tool
 
-## Build the solution.
+## Installation
 
-```dotnet build Bulldog.sln```
+Add the Bulldog package to your project:
 
-This project builds with [Nuke](https://nuke.build/).
+```bash
+dotnet add package Bulldog
+```
 
-To run the build call `build.cmd` which runs the FullBuild target of the nuke build [project](build/Build.csproj) (The same target that is run as part of CI)
+## Quick Start
 
-### Testing
+1. Create a new .NET project
+2. Add the Bulldog package
+3. Inherit from `ToolBase` in your main program class
+4. Implement your tool's logic
 
-Tests will be run as part of `FullBuild` target. To run them explicitly use:
+Example:
 
-```build.cmd Test```
+```csharp
+public class MyTool : ToolBase
+{
+    protected override async Task<int> RunAsync(CancellationToken cancellationToken)
+    {
+        // Your tool's logic here
+        return 0;
+    }
+}
+```
 
-Or unit tests and:
+## Building from Source
 
-```build.cmd SmokeTest```
+This project builds with [Nuke](https://nuke.build/).
 
-For the smoke tests.
+To build the solution:
+```bash
+dotnet build Bulldog.sln
+```
 
-## Usage
+To run the build:
+```bash
+build.cmd
+```
+This runs the FullBuild target of the nuke build [project](build/Build.csproj) (The same target that is run as part of CI)
 
-To use simply add a package reference to Bulldog package. The package defines a ToolBase class to inherit from which which handles the basics of:
+### Testing
+
+Tests will be run as part of `FullBuild` target. To run them explicitly:
 
-- Logging initialisation with Serilog
-- Dependency Injetion initialisation
-- Cancellation handling
-- CommandLine parsing with CommandlineParser library
+```bash
+build.cmd Test    # For unit tests
+build.cmd SmokeTest    # For smoke tests
+```
 
 The smoke test builds and runs an example [tool](tests/TestTool) which can be used as a reference.
 
-## Why
+## Contributing
+
+We welcome contributions! Take a look at the [issues](https://github.com/G-Research/Bulldog/issues) to find something to work on. Make sure you read the [CONTRIBUTING.md](CONTRIBUTING.md) before contributing.
```

Comment:
Let's remove the second sentence, it seems out of place. 
Replace "make sure you" with "Please" to sound more welcoming